src/components/autosuggest-material/autosuggest-material.tsx

```diff
@@ -20,61 +20,55 @@ const AutosuggestMaterial: React.FC<AutosuggestMaterialProps> = ({
   textDataLength
 }) => {
   const t = useText();
-  if (materialData.length < 1) {
-    return null;
-  }
   return (
-    <>
-      <li className="autosuggest__divider" />
-      <li>
-        <ul className="autosuggest__materials">
-          {/* eslint-disable react/jsx-props-no-spreading */}
-          {/* The downshift combobox works this way by design (line 50) */}
-          {/* incorrectIndex because in the whole of autosuggest dropdown it is
+    <li>
+      <ul className="autosuggest__materials">
+        {/* eslint-disable react/jsx-props-no-spreading */}
+        {/* The downshift combobox works this way by design (line 50) */}
+        {/* incorrectIndex because in the whole of autosuggest dropdown it is
           not the correct index for the item. We first need to add the length of
           items from autosuggest string suggestion to it for it to be accurate. */}
-          {materialData.map((item, incorrectIndex) => {
-            const index = incorrectIndex + textDataLength;
-            const authors: string[] = [];
-            item.work?.creators.forEach((creator) => {
-              authors.push(creator.display);
-            });
-            return (
-              <li
-                className={`autosuggest__material ${
-                  highlightedIndex === index
-                    ? "autosuggest__material--highlight"
-                    : ""
-                }`}
-                key={item.work?.workId}
-                {...getItemProps({ item, index })}
-              >
-                {/* eslint-enable react/jsx-props-no-spreading */}
-                <div className="autosuggest__material__content">
-                  <div className="autosuggest__cover">
-                    {item.work && (
-                      <Cover
-                        animate
-                        size="xsmall"
-                        pid={item.work.manifestations.first.pid as Pid}
-                      />
-                    )}
+        {materialData.map((item, incorrectIndex) => {
+          const index = incorrectIndex + textDataLength;
+          const authors: string[] = [];
+          item.work?.creators.forEach((creator) => {
+            authors.push(creator.display);
+          });
```

Comment:
Instead of using `let`and `forEach` you can also use `reduce`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce
It is a bit tricky to get used to but it is more cleanj I think.

Reply:
I am hereby using a general helper function we call flattenCreators :) 
Thanks for pointing it out ❤️